tests/plotting/test_texture.py

```diff
@@ -224,42 +223,13 @@ def test_texture_coordinates():
     # now grab the texture coordinates
     foo = mesh.active_t_coords
     assert np.allclose(foo, t_coords)
-    texture = pv.Texture(examples.mapfile)
-    with pytest.warns(PyVistaDeprecationWarning):
-        mesh.textures['map'] = texture
-    with pytest.warns(PyVistaDeprecationWarning):
-        assert mesh.textures['map'] is not None
-    with pytest.warns(PyVistaDeprecationWarning):
-        mesh.clear_textures()
-    with pytest.warns(PyVistaDeprecationWarning):
-        assert len(mesh.textures) == 0
 
 
 def test_multiple_texture_coordinates():
     mesh = examples.load_airplane()
     mesh.texture_map_to_plane(inplace=True, name="tex_a", use_bounds=False)
     mesh.texture_map_to_plane(inplace=True, name="tex_b", use_bounds=True)
     assert not np.allclose(mesh["tex_a"], mesh["tex_b"])
-    texture = pv.Texture(examples.mapfile)
-    with pytest.warns(PyVistaDeprecationWarning):
-        mesh.textures["tex_a"] = texture.copy()
-    with pytest.warns(PyVistaDeprecationWarning):
-        mesh.textures["tex_b"] = texture.copy()
-    with pytest.warns(PyVistaDeprecationWarning):
-        mesh._activate_texture("tex_a")
-    assert np.allclose(mesh.active_t_coords, mesh["tex_a"])
```

Comment:
Do we need to throw out the `active_t_coords` checks here? We still have the functionality, so how does one activate them?

Reply:
Oh whoops. Good catch